packages/fast-components-react-base/src/horizontal-overflow/examples.data.tsx

```diff
@@ -182,6 +184,10 @@ const examples: ComponentFactoryExample<HorizontalOverflowHandledProps> = {
             },
             ...images,
         ],
+        // Example of onHorizontalOverflowChange usage
+        // onHorizontalOverflowChange: (obj: HorizontalOverflowChange): any => (console.log("onHorizontalOverflowChange: ", obj)),
+        // Example of onScrollChange usage
+        // onScrollChange: (obj: ScrollChange): any => (console.log("onScrollChange: ", obj)),
```

Comment:
can this be in the README.md instead, noone is going to come to the example files for this information.

Reply:
> noone is going to come to the example files for this information.

I actually strongly _disagree_ with this, but I will move to the README for the sake of not contending. The first place I look when trying to learn implementation is the examples.